cmd/sonarExecuteScan.go

```diff
@@ -127,6 +132,10 @@ func handlePullRequest(options sonarExecuteScanOptions) error {
 			sonar.addOption("sonar.pullrequest.branch=" + options.ChangeBranch)
 			sonar.addOption("sonar.pullrequest.provider=" + provider)
 		}
+	} else {
+		if len(options.BranchName) > 0 {
```

Comment:
Minor: Since you already check an option on the outer if level, you could combine this into an "else if", no?